lnst/Recipes/ENRT/ConfigMixins/EnableIntelSSTTFMixin.py

```diff
@@ -0,0 +1,110 @@
+import re
+
+from lnst.Common.LnstError import LnstError
+from lnst.Common.Parameters import BoolParam
+from lnst.Controller.RecipeResults import ResultLevel, ResultType
+from lnst.Recipes.ENRT.ConfigMixins import BaseSubConfigMixin
+
+
+class EnableIntelSSTTFMixin(BaseSubConfigMixin):
+    enable_intel_sst_tf = BoolParam(default=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if getattr(self.params, "disable_turboboost", True):
+            raise ValueError(
+                "turboboost needs to be enabled when enabling intel_sst. "
+                f"intel_sst just allows higher clock speed for turbo boost."
+            )
+
+    @property
+    def intel_sst_tf_host_list(self):
+        """
+        The value of this property is a list of hosts for which the intel
+        SST-TF should be turned on. Derived class can override this property.
+        """
+        return self.matched
+
+    def _is_sst_supported(self, host):
+
+        sst_info = host.run("intel-speed-select --info", job_level=ResultLevel.DEBUG)
```

Comment:
is this available by default? if not maybe this should be preceded by https://github.com/LNST-project/lnst/blob/master/lnst/Common/Utils.py#L308